Integrations/integration-EWSv2.yml

```diff
@@ -185,6 +185,11 @@ script:
             return dict((k, v) for k, v in d.items() if v is not None)
         return d
 
+    def get_attachment_name(attachment_name):
+        if not str(attachment_name):
```

Comment:
@BenJoParadise if attachment_name is None this will cause it to become 'None' and then the condition would be false instead of true.

I think a better solution here would be something like:
`if not (attachment_name == 0 and attachment_name):`